<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dlg_set_spo2</class>
 <widget class="QDialog" name="Dlg_set_spo2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>血氧设置</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>脉搏音量</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>65</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>灵敏度</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_cancel">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>250</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>取消</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_maiboyintiaozhi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>195</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="m_btn_ok">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>99</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingsudu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形模式</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_maiboyinliang">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>20</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>11</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>135</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>脉率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_mailv">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>125</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_boxingmoshi">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>2</number>
   </property>
  </widget>
  <widget class="QComboBox" name="m_cmbb_lingmindu">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>55</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="maxVisibleItems">
    <number>3</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>205</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>脉搏音调制</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>波形速度</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
